src/utils/file-processor.test.js

```diff
@@ -0,0 +1,28 @@
+import CryptoJS from "crypto-js";
+
+import FileProcessor from "./file-processor";
+
+test("Creates correct chunks from file", () => {
+  const handle = "super-secret-key";
+  const file = new File(["test123-".repeat(2187)], "testFile.txt");
+  const chunks = FileProcessor.fileToChunks(file, handle).then(chunks => {
+    console.log("=======================");
+    // console.log(chunks);
+    console.log("=======================");
+  });
+
+  console.log("RABBIT");
+  // const encrypted = CryptoJS.Rabbit.encrypt("Message", "Secret Passphrase");
+  // const decrypted = CryptoJS.Rabbit.encrypt(encrypted, "Secret Passphrase");
+
+  var encrypted = CryptoJS.Rabbit.encrypt(
+    "Message1234",
+    "Secret Passphrase"
+  ).toString();
+  var decrypted = CryptoJS.Rabbit.decrypt(encrypted, "Secret Passphrase");
```

Comment:
@rfornea @automyr I'm not a crypto expert so this might be a newb question:

When I supply the "Wrong Passphrase" it doesn't return null, instead it gives me back garbage data. I thought we would determine if it is a treasure chunk if it returns null?

Reply:
When decrypting on the brokernode, it returns "" if the decrypting didn't work. Not sure why that isn't the case here...

Reply:
The broker is using AES I think, maybe different algorithms give different results when the decrypting doesn't work.